README.md

```diff
@@ -4,20 +4,16 @@ This is the Kids First RNA-Seq pipeline, which includes fusion and expression de
 
 ![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
 
-
-
-
-
 ## Introduction
 This pipeline utilizes cutadapt to trim adapters from the raw reads, if necessary, and passes the reads to STAR for alignment.
 The alignment output is used by RSEM for gene expression abundance estimation.
 Additionally, Kallisto is used for quantification, but uses pseudoalignments to estimate the gene abundance from the raw data.
 Fusion calling is performed using Arriba and STAR-Fusion detection tools on the STAR alignment outputs.
 Metrics for the workflow are generated by RNA-SeQC.
-**A new feature added as of February 27, 2020**, is an added workflow to run the artifact filtering portion of [annoFuse](https://www.biorxiv.org/content/10.1101/839738v1).
-Developed collaboratively between the Center for Data Driven Discovery in Biomedicine (D3b) and the Alex's Lemonade Stand Childhood Cancer Data Lab, this package adds annotation to arriba results, artifact filtering, and removes low-confidence fusion calls, as described in the paper.
-There are additional filtering capabilities provided beyond the lightweight workflow described in the [D3b annoFuse Workflow](#d3b-annofuse-workflow) section.
-**A new feature added as of May 21, 2020**, we now accept single end fastq inputs.
+
+Inputs from this can be used later on for further analysis in joint trio genotyping and subsequent refinement and deNovo variant analysis. If you would like to run this workflow using the cavatica public app, a basic primer on running public apps can be found [here](https://www.notion.so/d3b/Starting-From-Scratch-Running-Cavatica-af5ebb78c38a4f3190e32e67b4ce12bb).
```

Comment:
That first sentence should also go, just noticed that.

Reply:
Good catch. Removed.